doc/reference/std/sugar.md

```diff
@@ -39,13 +48,56 @@ finally-clause:
 
 Evaluates body with an exception catcher and an unwind finalizer.
 
+::: tip Examples:
+``` Scheme
+> (import :std/error)
+> (def (unbound-runtime-exception? e)
+    (match e ((RuntimeException exception: (? unbound-global-exception?)) #t)
```

Comment:
just `unbound-global-exception?` Does this for you.

Reply:
Not sure what you mean...

Reply:
the runtime already wraps the gambit exception predicates to unwrap them from RuntimeException.